build-artifacts/project-template-gradle/settings.gradle

```diff
@@ -5,3 +5,74 @@ include ':app'//, ':runtime', ':runtime-binding-generator'
 //project(':runtime-binding-generator').projectDir = new File("${System.env.ANDROID_RUNTIME_HOME}/test-app/runtime-binding-generator")
 
 file("google-services.json").renameTo(file("./app/google-services.json"))
+
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+
+import java.nio.file.Paths
+
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+import groovy.json.JsonSlurper
+import static org.gradle.internal.logging.text.StyledTextOutput.Style
+
+def USER_PROJECT_ROOT = "$rootDir/../../"
+def outLogger = services.get(StyledTextOutputFactory).create("colouredOutputLogger")
+def ext  = {
+    appResourcesPath = getProperty("appResourcesPath")
+    appPath = getProperty("appPath")
+}
+
+def getAppPath = { ->
+      def relativePathToApp = "app"
+      def nsConfigFile = file("$USER_PROJECT_ROOT/nsconfig.json")
+      def nsConfig
+
+      if (nsConfigFile.exists()) {
+          nsConfig = new JsonSlurper().parseText(nsConfigFile.getText("UTF-8"))
```

Comment:
Isn’t this a TypeScript file normally? Or am I confusing it with another one?

Reply:
It's a `.json` file that is i think deprecated now in favor of `nativescript.config.ts`. Both are seperate files, not to be confused as one.